system-addon/content-src/components/Search/Search.scss

```diff
@@ -0,0 +1,56 @@
+.search-wrapper {
+  cursor: default;
+  display: flex;
+  position: relative;
+  margin: 0 0 $search-margin-bottom;
+  width: 100%;
+  height: $search-height;
+
+  .search-label {
+    background: $search-glyph-image no-repeat center center / $search-glyph-size;
+    position: absolute;
+    top: 0;
+    offset-inline-start: 0;
+    height: 100%;
+    width: $search-input-left-label-width;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    z-index: 2;
+  }
+
+  input {
+    @include search-input;
+    padding-inline-start: 35px;
+
+    &:focus + button {
+      z-index: 1;
+      border-color: $search-blue;
+      transition: box-shadow 150ms;
+      box-shadow: $search-shadow;
+      background-color: $search-blue;
+      background-image: url('#{$image-path}glyph-forward-white.svg');
+      color: $white;
+    }
+  }
+
+  button {
```

Comment:
I noticed the button doesn't have a pointer cursor that the existing about:home/newtab have. A minor parity issues to check with UX?